corehq/apps/domain/management/commands/first_superuser.py

```diff
@@ -0,0 +1,21 @@
+import sys
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    help = "Return the first superuser, or a non-zero result."
+
+    def handle(self, **options):
+        if not settings.ALLOW_MAKE_SUPERUSER_COMMAND:
+            self.stdout.write('This command is not applicable to this '
+                              'environment.')
+            sys.exit()
+
```

Comment:
AFAIK, this setting isn't a _read_ access control, so I don't think this management command needs to check it. If the setting's value could prevent a `make_superuser` command later, it's cleaner to let that command fail (or handle it) if/when that happens rather than having special preemptive logic here.

```suggestion
```